nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/SFTPTransfer.java

```diff
@@ -642,6 +735,38 @@ public Socket createSocket(InetAddress inetAddress, int i, InetAddress inetAddre
         return sftpClient;
     }
 
+    void restrictSSHOptions(Config config) {
+        final String csvRegex = " *, *";
```

Comment:
Recommend replacing the pattern and `split()` with Spring Framework's `StringUtils.commaDelimitedListToSet()`

Reply:
I'd prefer not to introduce an external library dependency for something so trivial.

Reply:
That's a good point given that spring-core is not already a dependency.  In that case, I recommend promoting this variable to a static final at the class level.

Reply:
I actually got rid of the variable altogether and made a change. Commit coming shortly.